testing/scenario_app/lib/main.dart

```diff
@@ -40,7 +43,41 @@ void _handlePlatformMessage(String name, ByteData data, PlatformMessageResponseC
       data.setUint8(0, candidateScenario == null ? 0 : 1);
       callback(data);
     }
+  } else if (name == 'write_timeline') {
```

Comment:
I noticed that the PR description says that Dart profiling is disabled by default due to Firebase limitations. Does that mean that this code only writes some vm service events? Or does that mean that this is not executed at all on Firebase?

Reply:
This will pull out the JSON responses from the observatory, but right now they're not useful.  It just dumps them to logcat at the moment.